MergerService/appsettings.json

```diff
@@ -1,15 +1,19 @@
 {
   "GENERAL": {
-    "validate": true
+    "validate": false,
+    "outputPath": "watch"
   },
   "TASK": {
+    "jobManagerUrl": "http://localhost:8081",
+    "overseerUrl": "http://localhost:8080",
     "types": [
       {
-        "taskType": "",
-        "jobType": ""
+        "JobType": "Ingestion_New",
```

Comment:
Does every job type have the same task type...? If so, there is no need to write it explicitly.

Reply:
- this config change isn't relevant to this PR, its way too large already, especially since it might include another notification logic
-explicit has less chances of taking the wrong thing on future update